stage1.js

```diff
@@ -0,0 +1,233 @@
+const myStreet = 'Pushkina'
+const numBuilding = 65
+const isAdress = true
+
+// Выполнить сложение различных типов(string+boolean, string+number, number+boolean) 
+
+console.log(myStreet + isAdress)
+console.log(myStreet + numBuilding)
+console.log(numBuilding + isAdress)
+
+//Выполнить умножение различных типов(string * boolean, string * number, number * boolean)
+
+console.log(myStreet * isAdress)
+console.log(myStreet * numBuilding)
+console.log(numBuilding * isAdress)
+
+//Выполнить деление различных типов(string/boolean, string/number, number/Boolean)
+
+console.log(myStreet / isAdress)
+console.log(myStreet / numBuilding)
+console.log(numBuilding / isAdress)
+
+//На склад принимают груши и яблоки, напишите функцию, которая возвращает результат сложения
+// количества принятых груш и яблок
+//1:
+function sumFruits(apples, pear) {
+  return apples + pear
+}
+const takenFruits = sumFruits(50, 30)
+console.log(takenFruits);
```

Comment:
Please follow unified code style: pick up to use or not to use semicolons at the end of the line. Please check whole file.